portfolio/src/main/java/com/google/sps/servlets/NewTaskServlet.java

```diff
@@ -30,6 +30,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
             .build();
     datastore.put(taskEntity);
 
+    String textValue = request.getParameter("title");
+
+    System.out.println("You submitted: " + textValue);
+
+    response.getWriter().println("You submitted: " + textValue);
+
```

Comment:
What's the purpose of adding these lines?

Reply:
To show the user the message they typed

Reply:
Okay. I have a few notes on this 
- You already have a variable called `title` that holds the title parameter, so it will be better to reuse that variable for printing to the response. Especially since `title` has been sanitized with Jsoup while `textValue` is the raw input, which could contain javascript that you don't want to run.
- The line `System.out.println("You submitted: " + textValue);` just prints to the logs. Users generally don't see that. Is that necessary?
- The response to this POST request is never used as far as I can tell, so writing to the response won't show the response to the user. I could be wrong about that though; did you test it and confirm that it renders on the web page?